using_the_compiler/README.md

```diff
@@ -1,136 +1,322 @@
-# Using the compiler
+# Using the Compiler
```

Comment:
I'm not a fan of capitalizing all the words in titles, can we roll back those changes?

Reply:
It's a matter of style convention. I prefer title case but don't care about it that much.

Reply:
Yeah, absolutely! Still it'd be good to stick to one style and most of the current book conforms to "Using the compiler" rather than "Using the Compiler".